app/models/katello/content_view_puppet_environment.rb

```diff
@@ -34,8 +34,12 @@ def content_type
       Repository::PUPPET_TYPE
     end
 
-    def puppet?
-      true
+    def unprotected
+      false
+    end
+
+    def checksum_type
+      nil
     end
```

Comment:
@jlsherrill I'm not too sure what I should do about these unprotected and checksum type methods, any suggestions?